### ISSUE TITLE
IE11 compatibility

### ISSUE THREAD

**Issue by bobvandevijver** [dhershman1/vue-debounce#26]
The following line breaks in IE11:
https://github.com/dhershman1/vue-debounce/blob/873bb62728c2ab6fb5e86a44fec9e39c8a8c5a82/src/debounce.js#L2

Seems that this type of array assignment is a bit to much for it. Are you open for updating this to something a bit more old school, or do you know of some method to fix this?

**Comment by dhershman1** [comment]
Hmmm, I am already running it through buble during build so if you're using the dist version that should be IE11 compatible.

Are you trying to use debounce itself, or the directive?

**Comment by bobvandevijver** [comment]
Right, I'm not using the dist version, so that would explain it perfectly. I'm directly importing the `debounce.js` file: `import debounce from 'vue-debounce/src/debounce';`. That also means that it is something I should be able to fix from my side, by either including the compiled version, or fixing my own build system (man, why are all browser so different...).

The transpiled function you've post is indeed fully compatible with IE11 (just tested), so I will close this issue as solved 👍 

For the off-topic question: when you place the correct url directly in the editor, Github will convert it for you (see the example below). I got it by navigating to the relevant file, click on the corresponding line, click the menu on the left (with the three dots) and finally click the "Reference in new issue". 😃 
```
https://github.com/dhershman1/vue-debounce/blob/873bb62728c2ab6fb5e86a44fec9e39c8a8c5a82/src/debounce.js#L2
```

**Comment by bobvandevijver** [comment]
Seems like my Babel configuration is simply ignoring this line and keeps it as is... Will open an issue with Babel for it!

**Comment by dhershman1** [comment]
I'm using buble to compile mine, unless you mean your trying to have babel compile the source on your end?

**Comment by bobvandevijver** [comment]
Okay, I feel dumb now 😄 

So, I had the `import debounce from 'vue-debounce/src/debounce.js'` in my Vue component, which means that I indeed need to compile the source from my end. I was assuming that if I did that, Babel would take care of the imported source, and applying my own configuration successfully to it.

However, Babel by default ignores everything coming from the `node_modules` directory (https://github.com/symfony/webpack-encore/issues/670#issuecomment-555660424), even when imported in you own sources. Meaning that it was indeed not transpiled by Babel, resulting in errors on IE11. 

By either configuring Babel to do include this module, or by using the dist/module import (`import {debounce} from 'vue-debounce'` or `import debounce from 'vue-debounce/dist/debounce.min.js'`), it is really solved. Now I just need to fix the typescript typing :)
